Tests fail with sklearn 0.24.1 because of deprecated imports

Hello, quick friendly note:

I noticed on the NixOS continuous integration service, hydra, that the mlxtend tests fail to run with sklearn 0.24.1, the latest release (January 2021).

This is the error message:
```
==================================== ERRORS ====================================
________ ERROR collecting mlxtend/plotting/tests/test_pca_corr_graph.py ________
ImportError while importing test module '/build/source/mlxtend/plotting/tests/test_pca_corr_graph.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/fg7bbgshp0zadayr28yhsmrrivl500vi-python3-3.7.9/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
mlxtend/plotting/tests/test_pca_corr_graph.py:4: in <module>
    from sklearn.decomposition.pca import PCA
E   ModuleNotFoundError: No module named 'sklearn.decomposition.pca'
```

And here are the full log files: https://nix-cache.s3.amazonaws.com/log/3yyp38vdgk6jr62kf70zbvvln4crr0q3-python3.7-mlxtend-0.17.3.drv

In older versions of sklearn, I see this warning:
```
>>> import sklearn.decomposition.pca
/nix/store/ry8029wz0sm3lbr12p3hgv2xnwk5wvhs-python3-3.8.5-env/lib/python3.8/site-packages/sklearn/utils/deprecation.py:143: FutureWarning: The sklearn.decomposition.pca module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.decomposition. Anything that cannot be imported from sklearn.decomposition is now part of the private API.
  warnings.warn(message, FutureWarning)
```
so likely the imports just need to be tweaked a bit.

--- a/mlxtend/plotting/tests/test_pca_corr_graph.py
+++ b/mlxtend/plotting/tests/test_pca_corr_graph.py
@@ -1,7 +1,14 @@
 import pytest
 from mlxtend.plotting import plot_pca_correlation_graph
 from mlxtend.data import iris_data
-from sklearn.decomposition.pca import PCA
+
+from distutils.version import LooseVersion as Version
+from sklearn import __version__ as sklearn_version
+
+if Version(sklearn_version) < '0.22':
+    from sklearn.decomposition.pca import PCA
+else:
+    from sklearn.decomposition import PCA
 
 
 def test_pass_pca_corr():
@@ -62,4 +69,3 @@ def test_not_enough_components():
         plot_pca_correlation_graph(X, variables_names=['1', '2', '3', '4'],
                                    X_pca=X_pca,
                                    explained_variance=eigen[:-1])
-        

